portfolio/src/main/webapp/index.html

```diff
@@ -1,117 +1,130 @@
+      <button id="theme-toggle">Switch theme</button>
+    </header>
+    <main>
+      <h2>Hi!</h2>
+      <p>
+        I'm a rising sophomore at Cornell, double majoring in electrical & computer engineering, and
+        computer science. I love cooking & baking, karate, VFX, cinematography, and of course,
+        coding.
+      </p>
+      <p>
+        I was born in Nigeria, and I've lived all over the U.S., although when people ask I tell
+        them
+        I'm from Seattle because I lived in Bellevue & Kirkland for about 9 years of my 19-year
+        life.
+      </p>
+      <h2>Coding</h2>
+      <p>
+        You want my resume? <a href="/dl/resume.pdf">Here you go</a>.
+      </p>
+      <h2>Cooking</h2>
+      <p>
+        I know, we're in quarantine, everyone is cooking and baking. But look at this:
+      </p>
+      <div class='cooking-dish'>
+        <img
+          src="/img/orange-chicken.jpg"
+          alt="Spicy orange chicken with butter rice"
+        >
+        <h3>Spicy orange chicken with butter rice</h3>
+        <section class='blurb'>
+          <p>
+            This is based on
+            <a href="https://www.youtube.com/watch?v=N_87Ow7HTdQ">Joshua Weissman's recipe</a>. I
+            don't have Serrano chiles, because my mom doesn't believe in them, but I was able to
+            compensate by adding just a little bit of habañero oil. I also didn't have enough
+            cornstarch to make as much as I was hoping to, but I was able to get away with this by
+            substituting the remainder with tapioca flour.
+          </p>
+        </section>
+      </div>
+      <div class='cooking-dish'>
+        <img
+          src="/img/cornbread.jpg"
+          alt="Maple syrup cornbread with pork sausage and barbecue sauce"
+        >
+        <h3>Maple syrup cornbread with pork sausage and barbecue sauce</h3>
+        <section class='blurb'>
+          <p>
+            I can't even remember which recipe I used for this one. I should really keep better
+            records. But that's not the point. The point is that this cornbread was soft, delicate,
+            and delicious, with a lightweight maple syrup flavour. If I were to make it again, I'd
+            go
+            harder on the maple syrup, and also try adding jalapeños.
+          </p>
+        </section>
+      </div>
+      <div class='cooking-dish'>
+        <img
+          src="/img/raspberry-cheesecake.jpg"
+          alt="Raspberry swirl cheesecake"
+        >
+        <h3>Raspberry swirl cheesecake</h3>
+        <section class='blurb'>
+          <p>
+            I made this one by synthesizing these two recipes for cheesecake
+            (<a href="https://bakerbynature.com/the-best-new-york-style-cheesecake/">one</a>,
+            <a href="https://www.marthastewart.com/340227/raspberry-swirl-cheesecake">two</a>),
+            relying mostly on the second one. However, I did not make the raspberry sauce on the
+            spot,
```

Comment:
nit: it looks like the text here got put on a new line, but can be moved back up (same with paragraphs below)